### PR TITLE
{devel}[foss/2023b] NLTK v3.8.1, tqdm v4.66.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3154.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/00eda09074bfc651713b3e9e9c9bf5d6 for a full test report.

File: easybuild/easyconfigs/t/tqdm/tqdm-4.66.2-GCCcore-13.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+homepage = "https://github.com/tqdm/tqdm"
+description = """A fast, extensible progress bar for Python and CLI"""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['6cd52cdf0fef0e0f543299cfc96fec90d7b8a7e88745f411ec33eb44d5ed3531']
+
+builddependencies = [
+    ('binutils', '2.40'),
```
Comment (Micket) [inline easybuild/easyconfigs/t/tqdm/tqdm-4.66.2-GCCcore-13.2.0.eb:18]: was adding hatchling in 4.66.1 for GCCcore 12.3.0 a mistake?

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 5 out of 5 (2 easyconfigs in total)
alvis-skylake-build - Linux Rocky Linux 8.9, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/Micket/a38e961b058de348664c20b7ef12a416 for a full test report.

File: easybuild/easyconfigs/t/tqdm/tqdm-4.66.2-GCCcore-13.2.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/t/tqdm/tqdm-4.66.2-GCCcore-13.2.0.eb:18]: I guess it was necessary at the time, but not anymore thanks to #19777

We should probably revisit easyconfigs that have `hatchling` as a build dependency and clean them up, right @lexming?

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Going in, thanks @boegel!

File: easybuild/easyconfigs/t/tqdm/tqdm-4.66.2-GCCcore-13.2.0.eb (hunk 1 above)
Comment (Micket) [reply easybuild/easyconfigs/t/tqdm/tqdm-4.66.2-GCCcore-13.2.0.eb:18]: #20200

**Pull request merged**